boa/contracts/vyper/vyper_contract.py

```diff
@@ -155,6 +161,10 @@ def __init__(
                 msg += f"{capabilities.describe_capabilities()}"
                 raise Exception(msg)
 
+    @cached_property
+    def deployer(self):
```

Comment:
why would BaseVyperContract have this property?

Reply:
ah, both of the inheriting implementations have it. honestly, i think that VyperBlueprint should not have it, but that's out of scope of discussion for this PR